tests/unit/replicator_tests.py

```diff
@@ -116,6 +118,7 @@ def test_replication_with_generated_id(self):
         clone = Replicator(self.client)
         clone.create_replication(self.db, self.target_db)
 
+    @flaky(max_runs=3)
```

Comment:
Is there an alternative to fixing this test so we don't have to rely on `flaky`? Is it worth trying to adjust the N and Q values for the node? We've had this test pass consistently in the past.